app/assets/stylesheets/application.css

```diff
@@ -16,9 +16,14 @@
  *= require openseadragon.scss
  *
  *= require chf-overrides
- 
+ *
  *
  * Used by blacklight_range_limit
  *= require  'blacklight_range_limit'
  *
+ * TEMOPORARY -- fold this into work-show.scss before merging.
+ * Audio player:
+ *= require playlist.css
+ *=
```

Comment:
It should be in `stylesheets/local/playlist.scss` (note change suffix to `scss`, why not), and then it doens't need a `require`, it will get automatically picked up by our scss `@import` of everything in local (which happens to be in our local `sufia.scss` for confusing legacy reasons).